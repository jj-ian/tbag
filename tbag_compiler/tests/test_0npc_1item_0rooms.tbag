item {
	string name;
}

item Purse {
	name = "fluffy purse";
}

true {
	print("You have a ");
	print(Purse.name);

	endgame;
}
