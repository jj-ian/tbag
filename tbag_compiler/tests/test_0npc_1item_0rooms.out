You have a fluffy purse